website_event_require_login/controllers/main.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import http
+from odoo.addons.website_event.controllers.main import WebsiteEventController
+
+
+class RequireLoginToRegister(WebsiteEventController):
+
+    @http.route()
+    def registration_new(self, event, **post):
+        if event.website_require_login:
```

Comment:
Is there some reason why you aren't actually checking if the user is logged in or not?

Reply:
Sure :man_facepalming: